### ISSUE TITLE
Includes doesn't work properly

### ISSUE THREAD

**Issue by jaa2019** [C4sper2012/FourWheels#27]
```csharp
return await _dbContext.Arbejdsordrer
                .Include(m => m.Mekaiker)
                .Include(s => s.Servicetype)
                .Include(c => c.Bil)
                .ThenInclude(k => k.Kunde)
                .ToListAsync();
```

This doesn't include the `Mekaiker`and `Servicetype`- only `Bil`and `Kunde`is included

```csharp
[1] = Arbejdsordrer
 Bil = Bil
 BrugteReservdele = {List<BrugerReservdele>} null
 FKBilID = {int} 3
 FKBrugteReservedeleID = {int?} null
 FKMekanikerID = {int} 1
 FKServiceID = {int} 5
 Mekaiker = {Mekaiker} null
 Oprettet = {DateTime} 03/11/2022 09:55:41
 PKArbejdsordreID = {int} 2
 Servicetype = {Servicetype} null
 TidBrugt = {DateTime} 01/01/0001 00:00:00
 ```

**Comment by jaa2019** [comment]
This can be found on branch `Bugfix/Include_fail`

**Comment by jaa2019** [comment]
Can be closed with #28

**Comment by C4sper2012** [comment]
Issue was fixed in https://github.com/C4sper2012/FourWheels/pull/28
